src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -71,11 +71,13 @@ import { exec } from "child_process";
 export interface NetEmitterOptions {
     outputFile: string;
     logFile: string;
+    runCodegen: 'true' | 'false';
```

Comment:
Can it be boolean?

Reply:
No. Compiler takes it as string even you declare it as boolean.